src/courtroom.cpp

```diff
@@ -4052,6 +4054,7 @@ void Courtroom::on_effects_context_menu_requested(const QPoint &pos)
 {
   QMenu *menu = new QMenu();
```

Comment:
While you're at it, good practice to also parent these correctly, e.g.
```suggestion
  QMenu *menu = new QMenu(this);
```